packages/core/src/document.ts

```diff
@@ -153,6 +153,9 @@ export class Document {
 			thisProp.copy(otherProp, resolve);
 		}
 
+		// 5. Copy default scene
+		this.setDefaultScene(other.getRoot().getDefaultScene());
```

Comment:
With the suggestion below of making `setDefaultScene` public on the Root, this logic could be moved into `root.copy()`.